.env

```diff
@@ -91,3 +91,22 @@ BLACKFIRE_SERVER_TOKEN=<your_backfire_server_token>
 
 WEBHOOK_BASIC_AUTH_USERNAME=webhook
 WEBHOOK_BASIC_AUTH_PASSWORD=cCVMxdX8hfctDefk
+
+###> symfony/rate-limiter ###
+# Choose one of the storages below
+# RATE_LIMITER_STORAGE_DSN=redis://localhost/1
+# Inform the route names that you want to apply fixed window rate limit separeted by comma
+# RATE_LIMITER_FIXED_WINDOW_ROUTES="getAccessToken,refreshAccessToken,bulkCreateUsersAssignments"
```

Comment:
Not sure why but testing it with `RATE_LIMITER_FIXED_WINDOW_ROUTES="updateLineItems,getAccessToken,healthCheck,getUserAssignments"
`, only `getAccessToken` was not being restricted. Could you check it?

Reply:
Thanks @luisfn. Fixed.

Reply:
Reopening this one :)

I've tested tested again and the fix seems good, thanks. Just wondering, how about have a option to apply the rate limiter to `all` routes? Maybe a different option or the use of string all instead of the list. My concern is the size of the API since we have plans to expand it, seems good to have this option. WDYT?